source/install-atlas-cli.txt

```diff
@@ -210,77 +210,126 @@ Follow These Steps
+
+                           .. tab:: Ubuntu 20.04 (Focal)
+                              :tabid: ent-focal
+
+                              .. code-block:: sh
+
+                                 echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.com/apt/ubuntu focal/mongodb-enterprise/{+mdbVersion+} multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-enterprise.list
+
+                           .. tab:: Ubuntu 18.04 (Bionic)
+                              :tabid: ent-bionic
 
-                        .. code-block:: sh
+                              .. code-block:: sh
 
-                           echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.com/apt/ubuntu focal/mongodb-enterprise/5.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-enterprise.list
+                                 echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.com/apt/ubuntu bionic/mongodb-enterprise/{+mdbVersion+} multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-enterprise.list
 
-                     .. tab:: Ubuntu 18.04 (Bionic)
-                        :tabid: ent-bionic
+                     .. tab:: Debian
+                        :tabid: debian
 
-                        .. code-block:: sh
+                        Create a ``/etc/apt/sources.list.d/mongodb-enterprise.list`` file for your version of Debian.
+                        Replace ``{+mdbVersion+}`` with your edition of
+                        MongoDB.
 
-                           echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.com/apt/ubuntu bionic/mongodb-enterprise/5.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-enterprise.list
+                        .. tabs::
 
-                     .. tab:: Debian 10 (Buster)
-                        :tabid: ent-deb-10
+                           .. tab:: Debian 10 (Buster)
+                              :tabid: ent-deb-10
 
-                        .. code-block:: sh
+                              .. code-block:: sh
 
-                           echo "deb http://repo.mongodb.com/apt/debian buster/mongodb-enterprise/5.0 main" | sudo tee /etc/apt/sources.list.d/mongodb-enterprise.list
+                                 echo "deb http://repo.mongodb.com/apt/debian buster/mongodb-enterprise/{+mdbVersion+} main" | sudo tee /etc/apt/sources.list.d/mongodb-enterprise.list
 
-                     .. tab:: Debian 9 (Stretch)
-                        :tabid: ent-deb-9
+                           .. tab:: Debian 9 (Stretch)
+                              :tabid: ent-deb-9
 
-                        .. code-block:: sh
+                              .. code-block:: sh
 
-                           echo "deb http://repo.mongodb.com/apt/debian stretch/mongodb-enterprise/5.0 main" | sudo tee /etc/apt/sources.list.d/mongodb-enterprise.list
+                                 echo "deb http://repo.mongodb.com/apt/debian stretch/mongodb-enterprise/{+mdbVersion+} main" | sudo tee /etc/apt/sources.list.d/mongodb-enterprise.list
```

Comment:
**Question**: Same spacing as above between `debian` and `stretch`.

Reply:
Both should be correct based on existing docs